hibernate-osgi/src/test/resources/org/hibernate/osgi/test/testing-bundles.xml

```diff
@@ -8,5 +8,13 @@
     <feature name='hibernate-osgi-testing' version='5.0.0-SNAPSHOT'>
         <bundle>mvn:com.h2database/h2/1.3.170</bundle>
         <bundle>mvn:org.ops4j.pax.url/pax-url-aether/2.3.0</bundle>
+
+        <!-- org.jboss.spec.javax.transaction/jboss-transaction-api_1.2_spec does not include javax.transaction.xa package anymore -->
+        <!-- adding here bundles to provide the package -->
+        <bundle>mvn:javax.transaction/javax.transaction-api/1.2</bundle>
+        <bundle>mvn:javax.enterprise/cdi-api/1.2</bundle>
+        <bundle>mvn:javax.el/javax.el-api/3.0.0</bundle>
+
+        <bundle>mvn:com.sun.activation/javax.activation/1.2.0</bundle>
```

Comment:
Some questions here:
- `javax.transaction.xa` is present in Java SE now. I suppose that somehow Karaf doesn't expose it?
- wondering if we should depend on the `transaction-api` feature described here instead of declaring the dependencies manually: https://github.com/apache/karaf/blob/master/assemblies/features/enterprise/src/main/feature/feature.xml#L32 ?
- if it's not supposed to be optional, shoudn't this dependency be part of the core ORM feature instead?

Open questions, interested in your feedback :).

Reply:
Do we still support Java 7 or not?

Reply:
ORM 5.3 is targeting Java 8+

Reply:
So, may I remove some extra configurations for Java 7?

Reply:
It is a very good idea using `transaction-api` feature already provided by the Karaf container itself, but I think that we need to update the version, in fact we are using a very old Karaf version here: `3.0.3`.

Reply:
Outch, good catch. And a very old OSGi core and a very old pax-exam too.

We should certainly try to upgrade them if it's in the way of doing things properly.

Reply:
It is also needed to run JGroups 4 bundle. See https://groups.google.com/forum/#!topic/ops4j/mUZPQ1c-XTE.